README.md

```diff
@@ -340,13 +340,15 @@ Blog posts
 
 Sorted by recency:
 
+* [A deep dive into mutation testing and how the Mutant gem works][troessner]
```

Comment:
I know I'm late but just a small note: the other blog posts have the month and year in parens next to their title. Would be nice to keep this consistent. Helps convey whether the blog post links are stale or not